#!/bin/sh /usr/share/dpatch/dpatch-run
## 99-registered-version.dpatch by Fabian Greffrath <fabian@debian-unofficial.org>
##
## DP: The source code is setup to use shareware binaries. If you want to use
## DP: the registered version of ROTT, apply this patch and compile as normal.

@DPATCH@
diff -urNad rott-1.0+dfsg~/develop.h rott-1.0+dfsg/develop.h
--- rott-1.0+dfsg~/develop.h	2003-01-07 07:31:51.000000000 +0100
+++ rott-1.0+dfsg/develop.h	2008-01-10 09:28:18.000000000 +0100
@@ -40,8 +40,8 @@
 #define LOWCOST  0
 
 // Make sure only one of the following are on at one time
-#define SHAREWARE   1
-#define SUPERROTT   0
+#define SHAREWARE   0
+#define SUPERROTT   1
 #define SITELICENSE 0
 
 // cute little dopefish thing, only works with special patch?
